src/vidore_benchmark/retrievers/dse_qwen2_retriever.py

```diff
@@ -173,10 +173,13 @@ def get_scores(
         passage_embeddings: Union[torch.Tensor, List[torch.Tensor]],
         batch_size: Optional[int] = None,
     ) -> torch.Tensor:
+        """
+        Dot-product similarity between queries and passages.
+        """
         if isinstance(query_embeddings, list):
-            query_embeddings = torch.stack(query_embeddings).to(self.device)
+            query_embeddings = torch.stack(query_embeddings)
```

Comment:
I'm guessing the to self.device comes layer ?

Reply:
I have removed the `.to(self.device)` to make the `DSEQwen2Retriever` ISO with the others and because I think it's better to let the user the responsibility of sending the tensors fed to the `get_scores` method.

In particular, keeping `.to(self.device)` made my VM crashed for large datasets. For this reason, I have moved the computed embeddings to `cpu` in both the `forward_passages` and `forward_queries` method.